app/src/main/java/com/fightpandemics/home/HomeActivity.kt

```diff
@@ -17,10 +20,21 @@ class HomeActivity : BaseActivity(), HomeContract.View {
     @Inject
     lateinit var presenter: HomePresenter
 
+    private var isFabOpen = false
+
+    private lateinit var fab_open : Animation
+    private lateinit var fab_close : Animation
+    private lateinit var rotate_forward : Animation
+    private lateinit var rotate_backward : Animation
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_home)
         injectDependencies()
+        fab_open = AnimationUtils.loadAnimation(applicationContext, R.anim.fab_open)
+        fab_close = AnimationUtils.loadAnimation(applicationContext,R.anim.fab_close)
+        rotate_forward = AnimationUtils.loadAnimation(applicationContext,R.anim.rotate_forward)
+        rotate_backward = AnimationUtils.loadAnimation(applicationContext,R.anim.rotate_backward)
```

Comment:
To follow clean code practice, I'd suggest extracting these animations declaration in a function.